lib/iris/tests/results/unit/merge/ProtoCube/register__CubeSig/noise.txt

```diff
@@ -4,4 +4,4 @@ failed to merge into a single cube.
   cube.attributes keys differ: 'stuffed'
   cube.cell_methods differ
   cube.shape differs: (3,) != (2,)
-  cube data dtype differs: int64 != float64
\ No newline at end of file
+  cube data dtype differs: int64 != int8
```

Comment:
So this is fun: it looks like the changes in this PR have [fixed this test result](https://github.com/SciTools/iris/blob/master/lib/iris/tests/unit/merge/test_ProtoCube.py#L298) -- note how the dtype specified in the error text here now matches to the dtype _set in the test code_...